src/sentry/api/endpoints/organization_projects_sent_first_event.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.models import Project
+from sentry.api.base import DocSection
+from sentry.api.serializers import serialize
+from sentry.api.bases.organization import OrganizationEndpoint
+
+
+class OrganizationProjectsSentFirstEventEndpoint(OrganizationEndpoint):
+    doc_section = DocSection.ORGANIZATIONS
+
+    def get(self, request, organization):
+        """
+        Verify If Any Project Within An Organization Has Received a First Event
+        ```````````````````````````````````````````````````````````````````````
+
+        Returns true if any projects within the organization have received
+        a first event, false otherwise.
+
+        :pparam string organization_slug: the slug of the organization
+                                          containing the projects to check
+                                          for a first event from.
+        :auth: required
+        """
+        queryset = Project.objects.filter(organization=organization, first_event__isnull=False)
```

Comment:
I think this should only be the projects that a user has access to. Otherwise they'll go and try to configure the project and not be able to see it. cc @EvanPurkhiser

Reply:
It seems like the current logic has all projects (from organization details) being iterated on in the frontend. Not sure if this was intentional

Reply:
https://github.com/getsentry/getsentry/blob/master/static/getsentry/gsApp/components/trialBanner.jsx#L88

^ looks through all organization.projects

Reply:
Yeah, I just want to confirm with @EvanPurkhiser that this is actually intended behaviour. Other than that, this looks good!

Reply:
Yeah, if this is the current logic let's keep that.

Reply:
@davidenwang Hmm I think @wedamija is right? `organization.projects` is only projects a user has access to:

https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/organization.py#L209